packages/@tinacms/cli/src/index.ts

```diff
@@ -18,6 +18,7 @@ import { Command } from './command'
 import { baseCmds as baseCommands } from './cmds/baseCmds'
 import { logText } from './utils/theme'
 export { defineSchema } from './cmds/compile'
+export { TinaLevelClient } from './tinaLocalLevel'
```

Comment:
change the file name to `tinaLevel` or `tinaLevelClient`?

Reply:
Would it make more sense of this being in `@tinacms/graphql` ?

`@tinacms/cli` is typically a dev dependency which makes importing it sorta awkward (even if its only used locally)

Also, I was getting some Error when trying to import `@tinacms/cli` in pages/api/gql. (only in the mono repo and not in npm version).

Thought @kldavis4 and @jeffsee55 ?

Reply:
I updated it